auditbeat/Makefile

```diff
@@ -33,10 +33,15 @@ before-build:
 	     ${PREFIX}/${BEAT_NAME}-darwin.reference.yml
 
 	@cat ${ES_BEATS}/auditbeat/_meta/common.p1.yml \
-	     <(go run scripts/generate_config.go -os linux -concat) \
+	     <(go run scripts/generate_config.go -os linux -arch amd64 -concat) \
 	     ${ES_BEATS}/auditbeat/_meta/common.p2.yml \
 	     ${ES_BEATS}/libbeat/_meta/config.yml > \
 	     ${PREFIX}/${BEAT_NAME}-linux.yml
+	@cat ${ES_BEATS}/auditbeat/_meta/common.p1.yml \
+	     <(go run scripts/generate_config.go -os linux -arch i386 -concat) \
+	     ${ES_BEATS}/auditbeat/_meta/common.p2.yml \
+	     ${ES_BEATS}/libbeat/_meta/config.yml > \
+	     ${PREFIX}/${BEAT_NAME}-i386-linux.yml
 	@cat ${ES_BEATS}/auditbeat/_meta/common.reference.yml \
 	     <(go run scripts/generate_config.go -os linux -concat -ref) \
```

Comment:
We will also need to create an architecture specific "reference" config file for Linux too. (e.g. `${BEAT_NAME}-linux-i386.reference.yml`).